test/nni_test/nnitest/validators.py

```diff
@@ -12,6 +13,13 @@ class ITValidator:
     def __call__(self, rest_endpoint, experiment_dir, nni_source_dir, **kwargs):
         pass
 
+class ExportValidator(ITValidator):
+    def __call__(self, rest_endpoint, experiment_dir, nni_source_dir, **kwargs):
+        exp_id = osp.split(experiment_dir)[-1]
+        proc1 = subprocess.run(["nnictl", "experiment", "export", exp_id, "-t", "csv", "-f", "report.csv"])
+        assert proc1.returncode == 0, '`nnictl experiment export -t csv` failed with code %d' % proc1.returncode
```

Comment:
suggest to print out the content of csv and json files, so that we can check the result manually on pipeline when necessary.

Reply:
fixed in https://github.com/microsoft/nni/pull/2303/commits/d5951935b42303b2ca9512c62ec74c2c6af718f1